src/AppBundle/Entity/Repository/SurveyRepository.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace AppBundle\Entity\Repository;
+
+use AppBundle\Entity\Department;
+use AppBundle\Entity\Semester;
+use AppBundle\Entity\Survey;
+use AppBundle\Entity\User;
+use Doctrine\ORM\EntityRepository;
+
+class SurveyRepository extends EntityRepository
+{
+    /**
+     * @param User $user
+     * @param Department $department
+     * @param Semester $semester
+     * @return Survey[]
+     */
+    public function findOneByUserNotTaken(User $user, Semester $semester)
```

Comment:
Rename for å vise at det gjelder det gitte semesteret. `findOneByUserNotTakenInSemester` f.eks